meta-genivi-dev/recipes-support/lshw/files/cross-compile.patch

```diff
@@ -0,0 +1,39 @@
+---
+ src/Makefile      |    2 +-
+ src/core/Makefile |    2 +-
+ src/gui/Makefile  |    4 ++--
+ 3 files changed, 4 insertions(+), 4 deletions(-)
+
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -18,7 +18,7 @@ export MANDIR
+ export DATADIR
+ export SQLITE
+ 
+-CXX?=c++
++CXX?=$(CROSS_COMPILE)c++
```

Comment:
These changes are not part of this logical change (addition of lshw) even if they are required by lshw recipe. It's OK for now but for future reference, each logical change goes into it's own patch with it's own description and rationale.